docs/run/running/claiming-rewards.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 5
+description: Claiming rewards from your DV using the Launchpad.
+---
+# Claiming DV rewards
```

Comment:
Something went wrong here I think? This page is not related to claiming rewards, but a copy of monitoring.md

Reply:
This will be a brand new page that we need to write. I went ahead and created it so at least it exists, so then we can fill it in later. There are many such pages that need creation or overhauling once this PR is complete...

Reply:
Can we delete the monitoring part below then?

Reply:
ah, yes, didn't realize that. Must have been a mistake

Reply:
> There are many such pages that need creation or overhauling once this PR is complete...


Define complete. Should I be expecting them to be complete before we merge this PR as complete? Or are we taking them out and merging in a dedicated PR when ready?

Reply:
This PR is just the new structure. Once it is merged, I can get to work making new pages. I think this page is the only such "placeholder" page that I've created.